src/OpenTelemetry.SemanticConventions/scripts/semantic-conventions/generate.ps1

```diff
@@ -3,9 +3,9 @@ $SCRIPT_DIR=$PSScriptRoot
 $ROOT_DIR="${SCRIPT_DIR}/../../"
```

Comment:
There is also .sh script in the same folder. It should be modified in the same way.

Reply:
Will do.